panel/src/components/Collection/Items.vue

```diff
@@ -59,6 +60,29 @@
 
 <script>
 import { layout } from "@/mixins/props.js";
+
+export function findSelectedIndex(items, item) {
+	return items.findIndex((selected) =>
+		Object.hasOwn(selected, "_id")
+			? selected._id === item._id
+			: selected.id === item.id
+	);
```

Comment:
`Explanation:` To avoid the extra `batchEditingIdentifier` and having something similar for other use cases of `k-items`, I thought we could just check either `_id` or `id`.

Alternatively, one would have to pass the key as additional parameter to `findSelectedIndex` and then within `batchEditing.js` we would pass `batchEditingIdentifier`, in `k-items` we would use `id` in the `onSelected` method (and since batch editing keeps relying on just the `onSelect` method that would not conflict) .